Gemfile

```diff
@@ -1,3 +1,34 @@
 source 'https://rubygems.org'
 
 gemspec
+
+# rubocop:disable Bundler/DuplicatedGem
```

Comment:
RuboCop v0.58.2 has a false positive for this case.
So I had to disable `Bundler/DuplicatedGem`.
The latest RuboCop (v1.27.0) raises no offense.